### ISSUE TITLE
Only get errors on save

### ISSUE THREAD

**Issue by buggymcbugfix** [haskell/vscode-haskell#729]
I'm sure this must have been discussed somewhere before, alas I couldn't find such an issue here: is it possible to only get error messages on save or at least with some delay? I find it hard to concentrate when every keystroke causes big flashes of red.

**Comment by fendor** [comment]
Yeah, that should be possible to do. I am moving it to the main repo, I am not sure this is a vscode issue at the moment.
